lib/active_emoji/core_ext/thread.rb

```diff
@@ -13,4 +13,6 @@ class Thread
   alias ⛔❓ stop?
   alias 💹 value
   alias ⏰ wakeup
+  alias 🚪 exit
+  alias 💲 value
```

Comment:
Not sure about the :heavy_dollar_sign: as an alias for `Thread#value`. I’m not sure that translates well across regions, which is one of the stated goals of this project.